src/lib.rs

```diff
@@ -91,11 +92,14 @@ where
         self.prev_measurement = Some(measurement);
         let d = self.d_limit.min(d_unbounded.abs()) * d_unbounded.signum();
 
+        let output_unbounded = p + self.integral_term + d;
+        let output = self.limit.min(output_unbounded.abs()) * output_unbounded.signum();
+
         ControlOutput {
             p,
             i: self.integral_term,
             d,
-            output: (p + self.integral_term + d),
+            output: output
```

Comment:
Rust also supports the handy shorthand:
```suggestion
            output
```